boards/arm/bl5340_dvk/Kconfig.defconfig

```diff
@@ -27,6 +27,7 @@ endif # BOARD_BL5340_DVK_CPUAPP
 
 config BUILD_WITH_TFM
 	default y if BOARD_BL5340_DVK_CPUAPP_NS
+	depends on ZEPHYR_TRUSTED_FIRMWARE_M_MODULE
```

Comment:
These module dependencies look like they are also missing in `main`.

Can they be added to `main` and then backported for consistency?

Reply:
Nevermind - I don't see this used a lot on the main branch.

Reply:
Yep ... specially this one :)